### DISCUSSION TITLE
Website Redesign

### DISCUSSION THREAD

**Discussion by fmaclen** [jeremyevans/sequel#1863] [General]
Hey there, my background is primarily in design but I've been using Sequel at work and in personal projects increasingly.

I forked the repo and took at pass at a visual makeover of Sequel's website.
This is what it looks like: https://fmaclen.github.io/sequel/

**Changes:**
- Re-arranged navigation
- Added sitemap in the footer
- Adjusted font sizes to improve hierarchies and readability
- Added anchor tags to improve navigation within pages (specially the **Plugins** section)
- Replaced **Press** with a **Links** section
- Fixed broken links
- Refreshed project's branding with a new logo
- Added media queries to improve access on mobile devices

I hope this new design improves the UX for navigating the documentation and strengthens the message that this is an actively maintained project.

I'm open to doing a partial implementation of this design as well, for example: if you rather keep the old logo, only update some of the pages, etc.

If this change is welcome I'd be happy to submit a PR and outline more specific changes I made.

**Comment by jeremyevans** [comment]
This looks awesome, thank you so much for working on this!  I absolutely suck at visual design, so your assistance is very much appreciated.  Can you please submit a pull request for the changes?

**Reply by fmaclen** [reply]
Thanks Jeremy, PR submitted!

**Reply by burlesona** [reply]
Nice job @fmaclen!

**Reply by fmaclen** [reply]
@jeremyevans I'd recommend adding a bit more padding to the title in the homepage, specifically:

[`www/public/index.css`](https://github.com/jeremyevans/sequel/blob/master/www/public/css/index.css)
```css
div.index__hero {
        /* ... */
	padding-top: 96px;
	padding-bottom: 96px;
}
```

It'd look like this:

<img width="1359" alt="image" src="https://user-images.githubusercontent.com/1434675/166699157-10e9251c-9f48-439e-8eb8-39017dcef127.png">

I'd be happy to submit a PR if you want me to.

**Comment by jeremyevans** [comment]
@fmaclen I ended up removing a lot of the padding because it resulted in way too much whitespace, IMO.

**Reply by fmaclen** [reply]
Understood, no problem.
